.vscode/settings.json

```diff
@@ -1,3 +0,0 @@
-{
-    "python.pythonPath": ".venv/bin/python3"
```

Comment:
I think this file should be removed but I don't know how it got here in the first place 🤔